docs/specs/sections/definitions.tex

```diff
@@ -0,0 +1,98 @@
+\subsection{The Roles Involved} 
+
+\begin{itemize}
+    \item \textbf{User:} an entity that interacts with the wallet to request licenses and prove ownership of those.
+    \item \textbf{License Provider (LP):} an entity that receives requests for licenses, and upon acceptance, issues them.
+    \item \textbf{Service Provider (SP):} the entity that provides a service upon verification that a service request is correct. The SP may be the same as the LP entity or a different one.
+\end{itemize}
+
+\subsection{The Elements Involved} 
+
+\begin{itemize}
+    \item \textbf{Request:} a request includes the encryption of a stealth address belonging to the user, where the license has to be sent to, and a symmetric key. The structure is as follows:
+
+    \begin{center}
+        \begin{tabular}{ | p{0.15\linewidth} | p{0.2\linewidth} | p{0.55\linewidth} | } 
+        \hline
+        \textbf{Element} & \textbf{Type} & \textbf{Info.} \\
+        \hline
+        $(\rpk, R_{\req})$ & StealthAddress & It is a request stealth address of the LP. \\
```

Comment:
It is a request stealth address ~~of~~ **for** the LP.

Reply:
@xevisalle, maybe you were gramatically right with the use of the proposition "of", but I suggest using "for" because it captures the idea that the LP does not have a stealth address, but it is created by someone else _for_ the LP to use it.